scripts/setup-auth-qa.sh

```diff
@@ -13,8 +13,15 @@ fi
 
 QA="${PROJECT_NAME}QA"
 
+read -p "Enter login domain: (test.salesforce.com) " LOGIN_DOMAIN
+if [ -z $LOGIN_DOMAIN ] ; then
+    LOGIN_DOMAIN='test.salesforce.com'
+fi
+
+LOGIN_URL="https://$LOGIN_DOMAIN"
```

Comment:
@martinglauber Not only did I overlook this crucial line, but I committed the cardinal sin of testing with my eyeballs rather than pulling down this branch and actually running the script 😬 🤦‍♂️

My bad - all good for merge!